.github/workflows/pdf-build.yml

```diff
@@ -27,18 +28,21 @@ jobs:
             ${{ runner.os }}-node-
       - name: Install marp
         run: |
-          npm install --save-dev @marp-team/marp-cli
+          npm install --save-dev @marp-team/marp-cli markdown-pdf
       - name: Build PDFs
         run: |
-          for input in `grep -l 'marp: true' *.md */*.md`
+          for input in *.md */*.md;
           do
-            output=`echo -n $input | sed -e 's/.md$//g'`
-            echo "$output.pdf"
-            npx marp --allow-local-files "$input" -o "$output.pdf"
+            output=`echo -n $input | sed -e 's/\.md$/.pdf/g'`
+            echo "Input: $input, output $output";
+            if [[ $(grep -l 'marp: true' $input) ]];
+            then
+              npx marp --allow-local-files "$input" -o "$output"
+            else
+              npx markdown-pdf "$input" -o "$output"
```

Comment:
Does `markdown-pdf` support similar theming and formatting to `marp` or is it totally different?

Reply:
From what I've seen the formatting and syntax are fairly similar between the two.